controller_manager/src/controller_manager.cpp

```diff
@@ -318,7 +318,8 @@ controller_interface::return_type ControllerManager::configure_controller(
     RCLCPP_ERROR(
       get_logger(),
       "after configuring, controller %s is in state %s, expected Inactive",
-      controller_name.c_str());
+      controller_name.c_str(),
+      state.label().c_str());
```

Comment:
`new_state` is expected here...

```suggestion
      new_state.label().c_str());
```